src/Shell_Command.php

```diff
@@ -57,19 +58,33 @@ class Shell_Command extends EE_Command {
 	public function __invoke( $args, $assoc_args ) {
 
 		EE\Utils\delem_log( 'ee shell start' );
-		$args      = auto_site_name( $args, 'shell', '' );
-		$site_name = EE\Utils\remove_trailing_slash( $args[0] );
+		$global_services = [ 'global-nginx-proxy', 'global-db', 'global-redis' ];
+		$service         = \EE\Utils\get_flag_value( $assoc_args, 'service' );
```

Comment:
`use function EE\Utils\get_flag_value;` is already used on the top in the file.

```suggestion
		$service         = get_flag_value( $assoc_args, 'service' );
```